opengever/maintenance/scripts/bumblebee_installation.py

```diff
@@ -157,7 +166,15 @@ def main(app, argv=sys.argv[1:]):
                     LOG.info("Committing at {} documents.".format(index))
                     transaction.commit()
 
-        return transaction.commit()
+        transaction.commit()
+
+        if skipped:
+            LOG.warn('The following objects have been skipped:')
+            for obj_id, obj_url in skipped:
+                LOG.info("%s - %s" % (obj_id, obj_url))
```

Comment:
Can't use log level WARN for those because the log handler fuckery at the top of the module causes everything above INFO to be logged twice.